src/Alex/Utils/Texture2DJsonConverter.cs

```diff
@@ -56,6 +56,10 @@ public override void WriteJson(JsonWriter writer, Texture2D value, JsonSerialize
 		{
 			try
 			{
+				if (reader.Value == null)
```

Comment:
Has this ever occured? 0.o

Reply:
Yes, seems to happen 4 times in a row when you click muliplayer. I would rather not parse them when there is nothing to read, but the catch is handling it currently. Reader.Path = [0].CachedIcon, [1].CachedIcon and so on